crypto/init.c

```diff
@@ -675,3 +679,23 @@ int OPENSSL_atexit(void (*handler)(void))
 
     return 1;
 }
+
+#ifndef OPENSSL_SYS_VMS
+void OPENSSL_fork_prepare(void)
+{
+}
+
+/*
+ * WARNING!  If you put code in either OPENSSL_fork_parent or
+ * OPENSSL_fork_child, you MUST MAKE SURE that they are async-signal-
+ * safe.  See this link, for example:
+ *      http://man7.org/linux/man-pages/man7/signal-safety.7.html
+ */
+void OPENSSL_fork_parent(void)
+{
+}
+
+void OPENSSL_fork_child(void)
+{
+}
```

Comment:
A question on these functions...  is the intention that they should be reimplemented by the application that links with libcrypto?  In that case, you need to consider symbol clashes, and I would seriously implement them as callbacks, a little like in `mem.c`